src/Symfony/Component/DependencyInjection/ContainerInterface.php

```diff
@@ -21,7 +22,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
-interface ContainerInterface
+interface ContainerInterface extends PsrContainerInterface
```

Comment:
Not sure why php complains about it though : every call to test you can make on C, you can make on D, right?